packages/composables/src/mocks/Product.ts

```diff
@@ -466,98 +466,6 @@ const product: Product = {
   },
   isNew: false,
   calculatedMaxPurchase: 100,
-  //   seoCategory: {
```

Comment:
please remove the commented code if it's not relevant

Reply:
@mkucmus This is a `main` branch column :D